crates/sncast/src/starknet_commands/account/delete.rs

```diff
@@ -71,3 +73,17 @@ pub fn delete(
     let result = "Account successfully removed".to_string();
     Ok(AccountDeleteResponse { result })
 }
+
+pub async fn get_network_name(delete: &Delete, config: &CastConfig) -> Result<String> {
+    match (&delete.rpc, &delete.network) {
+        (None, None) | (Some(_), Some(_)) => {
+            bail!("Provide either `rpc` or `network`");
+        }
```

Comment:
Instead of this, how about we use clap to enforce only one of these arguments is always provided. It should be possible https://github.com/clap-rs/clap/blob/v3.1.2/examples/tutorial_builder/README.md#argument-relations